nvm.sh

```diff
@@ -2166,7 +2166,7 @@ nvm() {
           return 11
         fi
       fi
-      if [ -n "$NVM_USE_OUTPUT" ]; then
+      if [ -n "${NVM_USE_OUTPUT:-}" ]; then
```

Comment:
Thanks! I think this will work just as `"${NVM_USE_OUTPUT-}"` since there's no default value.